src/Relogify/Settings.fs

```diff
@@ -73,11 +73,11 @@ let performTransition (state: DialogState) (transition: Msg): DialogState * CmdM
 
     | (currentState, _disallowedTransition) -> currentState, []
 
-let initModel (applicationSettings: ApplicationSettings) =
-    let communityName = applicationSettings.CommunityName |> Option.defaultValue ""
-    let playerName = applicationSettings.PlayerName |> Option.defaultValue ""
-    { Settings = { CommunityName = communityName; PlayerName = playerName }
-      DialogState = if applicationSettings |> areSet then Closed else EditingCommunityName communityName }
+let initModel (communityNameHasBeenSaved: bool) =
+    if communityNameHasBeenSaved then
```

Comment:
Patter match? ;)

Reply:
I usually prefer `if/else` for a single boolean value (even though it's one extra line) :)